Add Postgres, MongoDB, RabbitMQ data services (#32)
portworx/pxdocs-backup: content/use-px-backup/backup-stateful-applications/mongodb.md

---
title: Backup MongoDB on Kubernetes
description: 

> keywords: 

Reviewer: ```suggestion
keywords: backup, mongodb
```

Author: done, for all data services pages now

weight: 4
hidesections: true
disableprevnext: true
---

> This page provides instructions for users on how to use pre and post backups rules with PX-Backup to achieve application-consistent backups for MongoDB on Kubernetes in production.

Reviewer: I think we've been using this verbiage for all of our stateful app intros, but it's not ideal. We should actually talk to the user directly when we write:

```suggestion
You can use the instructions on this page to create pre and post backup rules with PX-Backup, which take application-consistent backups for MongoDB on Kubernetes in production.
```

Author: done
